### PR TITLE
Add primary action button to PlantsPrimaryPage

### PR DESCRIPTION
**Pull request by karthikbtf**
- to support schedule observation use-case
- fixed storybook not loading due to `store` not defined
- added a story to capture the primary action button

PlantsPrimaryPageView - Action Button ⋅ Storybook 2023-09-26 10-39-15

### CODE REVIEW

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import type { Preview } from '@storybook/react';
 import { StyledEngineProvider, ThemeProvider } from '@mui/material';
 import { MemoryRouter } from 'react-router';
 import { Provider } from 'react-redux';
+import { store } from '../src/redux/store';
```
Comment (constanzauanini) [inline .storybook/preview.tsx:5]: Thanks for fixing it!!

**Comment by constanzauanini** [comment]
Should the button be at the same level of the dropdown for mobile? 
Screenshot 2023-09-26 at 10 36 05

**Comment by karthikbtf** [comment]
> Should the button be at the same level of the dropdown for mobile? Screenshot 2023-09-26 at 10 36 05

For mobile I think we want full width for the dropdown selector (the story doesn't capture what we want I think)
Terraware App 2023-09-26 19-08-01

**Comment by karthikbtf** [comment]
> > Should the button be at the same level of the dropdown for mobile? Screenshot 2023-09-26 at 10 36 05
> 
> For mobile I think we want full width for the dropdown selector (the story doesn't capture what we want I think) Terraware App 2023-09-26 19-08-01

I can move it up along with the title like we do for monitoring dashboard. Will do that as a follow up.

**Review by constanzauanini — APPROVED** [review]

**Comment by karthikbtf** [comment]
> > > Should the button be at the same level of the dropdown for mobile? Screenshot 2023-09-26 at 10 36 05
> > 
> > 
> > For mobile I think we want full width for the dropdown selector (the story doesn't capture what we want I think) Terraware App 2023-09-26 19-08-01
> 
> I can move it up along with the title like we do for monitoring dashboard. Will do that as a follow up.

I decided to clean it up here and moved it to the top in latest commit.
PlantsPrimaryPageView - Action Button ⋅ Storybook 2023-09-26 19-20-37

**Comment by constanzauanini** [comment]
> > > > Should the button be at the same level of the dropdown for mobile? Screenshot 2023-09-26 at 10 36 05
> > > 
> > > 
> > > For mobile I think we want full width for the dropdown selector (the story doesn't capture what we want I think) Terraware App 2023-09-26 19-08-01
> > 
> > 
> > I can move it up along with the title like we do for monitoring dashboard. Will do that as a follow up.
> 
> I decided to clean it up here and moved it to the top in latest commit. PlantsPrimaryPageView - Action Button ⋅ Storybook 2023-09-26 19-20-37

Thanks, I think it looks better now!

**Pull request merged**